lms/templates/certificates/certificate-base.html

```diff
@@ -0,0 +1,35 @@
+<%! from django.utils.translation import ugettext as _ %>
+<%! import mako.runtime %>
+<% mako.runtime.UNDEFINED = '' %>
+
+<%
+    # set doc language direction
+    from django.utils.translation import get_language_bidi
+    dir_rtl = 'rtl' if get_language_bidi() else 'ltr'
+%>
+
+<!DOCTYPE html>
+<html class="no-js" lang="en">
+    <head dir=-"${dir_rtl}">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        <title>${document_title}</title>
+
+        <!-- TODO: add description -->
```

Comment:
@mattdrayer, the format/length of the information would be 1-3 sentences about the view (this metadata is used in organic search results). I'm not sure what's appropriate as far as a default placeholder or an initial description from an MVP level.

@marcotuts, any suggestions or requirements for this now?